hw03_rileyj3.py

```diff
@@ -0,0 +1,111 @@
+######################################################################
+# Author: Bryce Riley
+# Username: rileyj3
+#
+# Assignment: HW03 : Fully Functional Gitty Psychedelic Robotic Turtles
+# Purpose: To create something complex and learn about github
+# Link to Document: https://docs.google.com/document/d/1EzPApStfIckqskXxnh8CNhMuNgis_GTE2FZZHHZvji4/edit?usp=sharing
+######################################################################
+
+import turtle
+
+wn = turtle.Screen()
+wn.bgcolor('turquoise')
```

Comment:
No code outside of functions! 

Move these two lines to the beginning of `main()`. If you have scoping issues, you need to pass the object into the functions using parameters:

```
def build_house(turt):
      ...
  
 build_house(t)
 ```

Reply:
Fixed. Still not too sure or positive on the parameters and how to use them, but to your "if", I don't think I ran into any issues. At least I hope not.